test/models/product_test.rb

```diff
@@ -94,4 +94,31 @@ class ProductTest < ActiveSupport::TestCase
       assert Product.of_working_set(item.name).count >= 0
     end
   end
+
+  test 'intervention_participations association' do
```

Comment:
Could you rename the test to be more descriptive? (something along the lines of "product has a way to get its intervention_participations" maybe ?)